src/app/content/studyGuides/components/StudyGuidesCTA/index.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import StudyGuidesCTA from '.';
+import createTestStore from '../../../../../test/createTestStore';
+import { receiveLoggedOut, receiveUser } from '../../../../auth/actions';
+import MessageProvider from '../../../../MessageProvider';
+import { Store } from '../../../../types';
+import * as Styled from './styles';
+
+describe('StudyGuidesCTA', () => {
+  let store: Store;
+
+  beforeEach(() => {
+    store = createTestStore();
+  });
+
+  it('does not render if user is logged in', () => {
+    store.dispatch(receiveUser({} as any));
+
+    const component = renderer.create(<Provider store={store}>
+      <MessageProvider>
+        <StudyGuidesCTA />
+      </MessageProvider>
+    </Provider>);
+
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAWrapper)).toThrow();
+  });
+
+  it('does render if user is not logged in', () => {
+    store.dispatch(receiveLoggedOut());
+
+    const component = renderer.create(<Provider store={store}>
+      <MessageProvider>
+        <StudyGuidesCTA />
+      </MessageProvider>
+    </Provider>);
+
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAWrapper)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAContent)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTATitle)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAButtons)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAButton)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTASeparator)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTALink)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAInfoWrapper)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAInfo)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAArrowDesktop)).not.toThrow();
+    expect(() => component.root.findByType(Styled.StudyGuidesCTAArrowMobile)).not.toThrow();
```

Comment:
I think you've suggested on another pr to check for specific elements rather than using snapshots because it is hard to follow what's missing after snapshot update. I think checking for specific items is okay, but yea, it's a long list.

Reply:
I did, but i feel like it's not necessary to assert every component because they are not all separately conditionally rendered. But you are right too, if not every element is asserted then its still possible to miss it after updating. 

I'm not sure what's the better approach then. Snapshots are usually brittle and easy to miss, assertions on each component are tedious. What do you think?

Reply:
I think it doesn't really matter, but snapshots have one more advantage - they show you a text inside components, aria labels, etc. So I think we can go with a snapshot in this one, especially that it will probably not be changing anytime soon